docs/product/stats/index.mdx

```diff
@@ -10,11 +10,87 @@ The "Usage" tab shows your organization’s event and attachments usage, while a
 
 ## Usage Stats
 
-The "Usage" tab shows you the events (errors and transactions) and attachments Sentry has received across your entire organization along with a list of all projects and their consumption of each of those. Your usage falls into one of three categories: _Accepted_, _Filtered_, or _Dropped_. Filtered events and attachments are ones that were blocked due to your [inbound data filter](/concepts/data-management/filtering/) rules. Dropped events and attachments are ones the system dropped or discarded due to rate limits, quotas, spike protection, [size limits](/concepts/data-management/size-limits), or because of invalid data.
-
-You can see which projects have had a recent spike or are the most active and may need more attention. On this page, you can also control the date range that’s displayed, allowing you to focus on a smaller period of time or zoom out for a longer view. The page is designed for org admins, but is available to all team members. This is helpful if a team member who isn't an admin is tasked with reviewing these statistics. Also, this allows team members associated with specific projects to use this page to investigate why their events are being dropped.
-
-![Overview of Usage Stats page](./img/usage-stats.png)
+The "Usage" tab provides an overview of the events (errors and transactions) and attachments that Sentry has received across your entire organization. It includes a detailed breakdown of each project's consumption categorized as _Accepted_, _Filtered_, _Discarded_, or _Dropped_.
+
+#### Accepted
+
+Events and attachments that were successfully processed and stored.
+
+#### Discarded
+
+Events and attachments discarded on the client side due to the sampling configuration in the SDK or other factors.
+
+#### Filtered
+
+| Reason                                                                              | Meaning                                                                            |
+| ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Browser Extensions](/concepts/data-management/filtering/#browser-extension-errors) | Filtered by browser extension.                                                     |
+| Denied Name                                                                         | Filtered due to name being denied.                                                 |
+| Disabled Namespace                                                                  | Filtered due to the namespace being disabled.                                      |
```

Comment:
These two are related to metrics and will be removed, so let's not list them here.
```suggestion
```